webpack.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
         { loader: 'file-loader', options: { name: 'css/[name].css' } }, // Extract css into seperate file
         'postcss-loader',
         'extract-loader',
-        'css-loader?-minimize', // translates CSS into CommonJS modules
+        'css-loader', // translates CSS into CommonJS modules
```

Comment:
?-minimize no longer needed? Will it still be minimized?

Reply:
It's deprecated, I think. Minimizing is still done by postcss cssnano. So this actually didn't do anything

Reply:
It better be 🙈